packaging/linux/Arch/PKGBUILD

```diff
@@ -1,6 +1,14 @@
 # Edit on github: https://github.com/LizardByte/Sunshine/blob/master/packaging/linux/Arch/PKGBUILD
 # Reference: https://wiki.archlinux.org/title/PKGBUILD
 
+## options
+: "${_run_unit_tests:=false}" # if set to true; unit tests will be executed post build; useful in CI
+: "${_use_cuda:=detect}" # nvenc
+: "${_cuda_gcc_version:=$(LC_ALL=C pacman -Si cuda | grep -Pom1 '^Depends On\s*:.*\bgcc\K[0-9]+\b')}"
+: "${_support_headless_testing:=false}"
```

Comment:
I believe these need to be added to the docker image, because it doesn't look like the unit tests are running.

We also want cuda and everything enabled for CI.

https://github.com/LizardByte/Sunshine/blob/master/docker/archlinux.dockerfile

Reply:
Done. `cuda` would have been active however since it was installed, but I still explicitly set it.